api/shared-modules/src/node/utils/encryption/index.ts

```diff
@@ -48,3 +49,21 @@ export const decrypt = (cipher: string, secret: string) => {
 	const decrpyted = Buffer.concat([decipher.update(Buffer.from(hash, 'hex')), decipher.final()]);
 	return decrpyted.toString();
 };
+
+export const asymEncrypt = (data: any, privateKey: string, channelPublicKey: string): string => {
+	const diffie = crypto.createDiffieHellman(bs58.decode(privateKey));
+	const sharedKey = diffie.computeSecret(bs58.decode(channelPublicKey))
+	const dataBytes = aesjs.utils.utf8.toBytes(JSON.stringify(data));
+	const aesCtr = new aesjs.ModeOfOperation.ctr(sharedKey);
+	const encrypted = aesCtr.encrypt(dataBytes);
```

Comment:
why cant we use the existing encrypt function here? can you please double check if it is possible